File: app-kotlin/src/main/java/com/TritiumGaming/phasmophobiaevidencepicker/activities/fragments/investigation/evidence/views/investigation/InvestigationSection.java
package com.TritiumGaming.phasmophobiaevidencepicker.activities.fragments.investigation.evidence.views.investigation;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.constraintlayout.widget.ConstraintLayout;

import com.TritiumGaming.phasmophobiaevidencepicker.R;

public class InvestigationSection extends ConstraintLayout {

    public InvestigationSection(@NonNull Context context) {
        super(context);
        initView();
    }

    public InvestigationSection(@NonNull Context context, @Nullable @org.jetbrains.annotations.Nullable AttributeSet attrs) {
        super(context, attrs);
        initView();
    }

    public InvestigationSection(@NonNull Context context, @Nullable @org.jetbrains.annotations.Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        initView();
    }

    public InvestigationSection(@NonNull Context context, @Nullable @org.jetbrains.annotations.Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
        initView();
    }

    private void initView() {
        inflate(getContext(), R.layout.item_evidence_tool_section, this);
    }

    public void setLabel(String label) {
        ((TextView)findViewById(R.id.label_container)).setText(label);
    }
}

File: app-view/src/main/java/com/tritiumgaming/phasmophobiaevidencepicker/presentation/ui/activities/investigation/evidence/common/investigation/section/InvestigationSection.kt
package com.tritiumgaming.phasmophobiaevidencepicker.presentation.ui.activities.investigation.evidence.common.investigation.section

import android.content.Context
import android.util.AttributeSet
import android.view.View
import android.view.ViewStub
import android.widget.TextView
import androidx.constraintlayout.widget.ConstraintLayout
import com.tritiumgaming.phasmophobiaevidencepicker.R

class InvestigationSection : ConstraintLayout {
    constructor(context: Context) :
            super(context) { initView() }

    constructor(context: Context, attrs: AttributeSet?) :
            super(context, attrs) { initView() }

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) :
            super(context, attrs, defStyleAttr) { initView() }

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defStyleRes: Int) :
            super(context, attrs, defStyleAttr, defStyleRes) { initView() }

    private fun initView() {
        inflate(context, R.layout.item_evidence_tool_section, this)
    }

    fun setLabel(label: String?) {
        (findViewById<View>(R.id.label_container) as TextView).text = label
    }

    fun getList(): ViewStub? {
        return findViewById(R.id.list)
    }
}
